### ISSUE TITLE
Play time cursed?

### ISSUE THREAD

**Issue by PaoAzeitado** [Extegral/Enigmatic-Legacy#489]
How do i modify the stat play time with and without seven curses?
When i try to modify it through the stats .json it goes back to what it was, while the other stats if modified don't go back.
Is it being stored somewhere?

**Comment by Sebaty5** [comment]
`saves/<world>/stats/<your_uuid>.json`. 
There you are looking for `play_time_with_seven_curses` and `play_time_without_seven_curses` stats, which record respective times in game ticks.

Taken from the pinned messages in the `help-and-questions` channel in the official discord.

**Comment by PaoAzeitado** [comment]
yes, i found that, but when i try to modify the `play_time_with_seven_curses` or the `play_time_without_seven_curses` they go back, let's say my `play_time_with_seven_curses` is '30000', if i change it to '1000', when i join the world it goes back to '30000' same with the `play_time_without_seven_curses` but that doesn't happen with the normal stats killed by zombies for example

**Comment by Sebaty5** [comment]
Single or multiplayer? I know that playerdate is stored differently for each of those.

but its best to wait for Aizistral to answer this question. I have never dealt with changing my times.

**Comment by PaoAzeitado** [comment]
Understandble, singleplayer using lan to play with a friend, do you happen to know when he's gonna be able to?

**Comment by Sebaty5** [comment]
All I can do is bring this to his attention...
When he has the time to check this out I do not know.

**Comment by copygirl** [comment]
When you play in singleplayer, the local player data is stored in the `level.dat` file.

**Comment by milkev** [comment]
i had to change it in the level.dat, player.dat and statistics.json all at the same time for it to change properly in singleplayer ,-,

**Comment by CraftyZombie** [comment]
This is no longer a problem, as the command '/setringtime' can do this operation properly.
